### DISCUSSION TITLE
Kind cluster Kong Ingress Controller 2.8.0 failure to get a peer from the ring-balancer

### DISCUSSION THREAD

**Discussion by vugardzhamalov** [Kong/kubernetes-ingress-controller#3310] [General]
Hi there!

I normally deploy Kong Ingress Controller on my Kind cluster via so called  dbless config provided [here](https://github.com/Kong/kubernetes-ingress-controller/blob/main/deploy/single/all-in-one-dbless.yaml) . It seems as if latest changes merged to master break my Ingress. So far I've confirmed that if I change controller back to version 2.7.0 [here](https://github.com/Kong/kubernetes-ingress-controller/blob/48117d106a0c0d81d47fc164b21abd94abc7f60a/deploy/single/all-in-one-dbless.yaml#L1712) - my Ingress does work as expected. 

Otherwise if I deploy controller v2.8.0 with the same Ingress object - I  end up seeing only one error:

`{"message":"failure to get a peer from the ring-balancer"}`

I was wondering if controller v2.8.0 requires further changes  made to Ingress? Thank you!

**Comment by vugardzhamalov** [comment]
Had another look at logs, I was observing something like this:

`level=error msg="resource processing failed: can't find port for backend kubernetes service: no suitable port found" GVK="/v1, Kind=Service" name=blablah`

So I've changed backend service port names to port numbers - it seems as if this has resolved the issue.

This brings me to another question: is this expected behavior under v2.8.0? Anyone knows if backend service port names are not supported anymore?

**Reply by pmalek** [reply]
@vugardzhamalov Are you using service port names as described in https://github.com/Kong/kubernetes-ingress-controller/issues/3296?

**Reply by vugardzhamalov** [reply]
Thank you @pmalek ! Yes I believe it is identical. Waiting for the next release  ;-)
